iocore/net/SSLNetVConnection.cc

```diff
@@ -1632,11 +1633,23 @@ SSLNetVConnection::callHooks(TSEvent eventId)
   this->serverName = const_cast<char *>(SSL_get_servername(this->ssl, TLSEXT_NAMETYPE_host_name));
   if (this->serverName) {
     auto *hs = TunnelMap.find(this->serverName);
+    if (!hs) {
+      Vec<cchar *> keys;
+      wildTunnelMap.TunnelhMap.get_keys(keys);
+      for (int i = 0; i < static_cast<int>(keys.length()); i++) {
+        std::string_view sv{this->serverName, strlen(this->serverName)};
```

Comment:
`std::string_view sv{this->serverName};` works fine if the argument is C string.